src/lib/uri/uri_parser.rl

```diff
@@ -101,7 +101,7 @@ uri_raw_parse(struct uri_raw *uri, const char *p)
 			>{ s = p; }
```

Comment:
The paths in the generated `uri_parser.c` may be important, see 415c05868f95b45a56904d5aa581358aced05efe. I guess they shouldn't be changed on the file regeneration.

Reply:
I replaced the paths with the correct ones.